<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    background-color: #505050;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;      /* Ярко-оранжевый, как на iPhone */
    color: white;                   /* Белый текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #ffa733;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #cc7a00;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;      /* Ярко-оранжевый, как на iPhone */
    color: white;                   /* Белый текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #ffa733;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #cc7a00;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;      /* Ярко-оранжевый, как на iPhone */
    color: white;                   /* Белый текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #ffa733;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #cc7a00;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;      /* Тёмно-серый фон */
    color: white;                   /* Белый текст */
    font-size: 28px;                /* Крупный шрифт */
    font-weight: bold;
    border: none;
    border-radius: 40px;            /* Круглая форма */
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #6b6b6b;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #323232;      /* Темнее при нажатии */
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;      /* Ярко-оранжевый, как на iPhone */
    color: white;                   /* Белый текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #ffa733;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #cc7a00;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #505050;
    color: white;
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    padding-left: 32px;    /* Сдвиг текста немного влево */
    padding-right: 32px;
    min-width: 20px;      /* Шире обычной */
    min-height: 80px;      /* Как у других кнопок */
}

QPushButton:hover {
    background-color: #6b6b6b;
}

QPushButton:pressed {
    background-color: #323232;
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    background-color:  #1e1e1e;         /* Тёмный фон */
    color: #f0f0f0;;                 /* Светло-серый/почти белый текст */
    font-size: 36px;                    /* Крупный шрифт */
    font-weight: bold;
    padding: 15px 20px;                 /* Отступы сверху/снизу и слева/справа */
    border-radius: 12px;                /* Скруглённые углы */
    border: 0px solid #333333;          /* Тонкая рамка */
    qproperty-alignment: 'AlignRight | AlignVCenter'; /* Выравнивание по правому краю и по центру по вертикали */
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #d4d4d2;      /* Светло-серый фон */
    color: black;                   /* Чёрный текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #e0e0de;
}

QPushButton:pressed {
    background-color: #bbbbbb;
}
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ff9500;      /* Ярко-оранжевый, как на iPhone */
    color: white;                   /* Белый текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #ffa733;      /* Светлее при наведении */
}

QPushButton:pressed {
    background-color: #cc7a00;      /* Темнее при нажатии */
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #d4d4d2;      /* Светло-серый фон */
    color: black;                   /* Чёрный текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #e0e0de;
}

QPushButton:pressed {
    background-color: #bbbbbb;
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusminus">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #d4d4d2;      /* Светло-серый фон */
    color: black;                   /* Чёрный текст */
    font-size: 28px;
    font-weight: bold;
    border: none;
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    padding: 0;
}

QPushButton:hover {
    background-color: #e0e0de;
}

QPushButton:pressed {
    background-color: #bbbbbb;
}
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
